Purchases/Attribution/RCAttributionFetcher.m

```diff
@@ -70,14 +70,16 @@ - (nullable NSString *)identifierForVendor {
 - (void)adClientAttributionDetailsWithCompletionBlock:(RCAttributionDetailsBlock)completionHandler {
     // Should match available platforms in
     // https://developer.apple.com/documentation/iad/adclient?language=objc
-    #if TARGET_OS_IOS
+    #if TARGET_OS_IOS || TARGET_OS_MACCATALYST
     RCAdClientProxy * _Nullable adClientProxy = [self.attributionFactory adClientProxy];
     if (!adClientProxy) {
         [RCLog warn:[NSString stringWithFormat:@"%@",
                      RCStrings.attribution.search_ads_attribution_cancelled_missing_iad_framework]];
         return;
     }
     [adClientProxy requestAttributionDetailsWithBlock:completionHandler];
+    #else
+    completionHandler([NSDictionary new], nil);
     #endif
```

Comment:
Well, I added this else block because I think if the code is not executed in iOS, this [closure](https://github.com/RevenueCat/purchases-ios/blob/3afe8959e15deae11a17c3d843b2075c948266ed/Purchases/Attribution/RCAttributionPoster.m#L118) never will be released. Maybe I'm wrong... 🤔